test/helper/cookie.ts

```diff
@@ -20,7 +20,7 @@ import { AUTHORIZATION_HEADER_NAME } from '../constant';
 
 export function extractAuthCookie(response: Response) {
   const setCookieHeaders = response.header['set-cookie'] as string[];
-  let securityAuthCookie: string;
+  let securityAuthCookie: string | null = null;
```

Comment:
Does `securityAuthCookie` need to be `null` here? The [return value](https://github.com/opensearch-project/security-dashboards-plugin/blob/07688fab15a6ddc2ccde86597144b67eab59d265/test/helper/cookie.ts#L30) can be `undefined`.

Reply:
yes, since null can valid value for securityAuthCookie